dart/lib/signalr_core/src/header_names.dart

```diff
@@ -0,0 +1,4 @@
+abstract class HeaderNames {
+  static String get authorization => 'Authoriation';
```

Comment:
```suggestion
  static String get authorization => 'Authorization';
```
Did this ever work? 😛